### PR TITLE
Css fixies

### PR DESCRIPTION
**Pull request by asherism**
mostly trades table styling like time format, external link icon, etc

### CODE REVIEW

**Comment by platschi** [comment]
There's a conflict with the TimeDisplay component we introduced this release, so the time styling should probably happen in there.

File: components/Currency/CurrencyPrice/CurrencyPrice.tsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ import { formatCurrency, FormatCurrencyOptions } from 'utils/formatters/number';
 
 import { ContainerRowMixin } from '../common';
 import { ethers } from 'ethers';
+import { composeInitialProps } from 'react-i18next';
```
Comment (koredefashokun) [inline components/Currency/CurrencyPrice/CurrencyPrice.tsx:13]: I think we can remove this line since we don't use it.

**Review by koredefashokun — APPROVED** [review]
Code changes look good.

**Comment by koredefashokun** [comment]
Just noticed @platschi's comment as well. Also a bit worried about the number of commits compared to the actual number of changes. Did we carry over some stuff from another branch?

**Comment by platschi** [comment]
> Just noticed @platschi's comment as well. Also a bit worried about the number of commits compared to the actual number of changes. Did we carry over some stuff from another branch?

Yeah lots of overlap with #806.  I guess we should make sure to set up new branches once a PR is submitted

**Comment by platschi** [comment]
> > Just noticed @platschi's comment as well. Also a bit worried about the number of commits compared to the actual number of changes. Did we carry over some stuff from another branch?
> 
> Yeah lots of overlap with #806. I guess we should make sure to set up new branches once a PR is submitted

@asherism Can you look at this?

**Review by platschi — APPROVED** [review]

**Pull request merged**